test/WasmSpec/rlexe.xml

```diff
@@ -1260,6 +1260,13 @@
       <tags>exclude_dynapogo</tags>
     </default>
   </test>
+  <test>
```

Comment:
In order to get this and not lose it next time I update the spec repo.
You have to add an entry in `test\WasmSpec\convert-test-suite\config.json` `"folders": [ ... "features/nontrapping" ]`
Then you need to run `node convert-test-suite` to generate rlexe.xml